k8s_test_harness/util/k8s_util.py

```diff
@@ -142,7 +144,7 @@ def wait_for_resource(
             resource_type,
             name,
             "--timeout",
-            "60s",
+            "1s",
```

Comment:
Actually, this should be `1min` I guess.

Reply:
I have “shifted” the wait from the ‘kubectl’ call to the retry function itself, with the total delay still amounting to more or less 1minute.

The ‘--timeout’ kwarg to ‘kubectl’ is still required though, otherwise we risk it waiting indefinitely…